_rules/autocomplete-valid-value-73f2c2.md

```diff
@@ -90,63 +86,47 @@ Single autocomplete term for select.
 
 #### Passed Example 3
 
-Autocomplete term, only valid for textarea.
+This `autocomplete` [attribute value][] only has the required token, and is valid for a `textarea` element. Mixing upper and lower case letters is allowed for `autocomplete` attributes.
 
 ```html
 <textarea autocomplete="Street-Address"></textarea>
 ```
 
 #### Passed Example 4
 
-Two autocomplete terms.
+This `autocomplete` [attribute value][] has a `work` token, allowed because it is used before `email`. The `email` token is allowed on `input` elements with a `type` [attribute value][] of `text`.
```

Comment:
I think sometimes the outgoing short phrases are a little clearer about certain things. In this case that this example has a list of terms in the attribute value ...

```suggestion
This `autocomplete` [attribute value][] list includes a `work` token, allowed because it is used before `email`. The `email` token is allowed on `input` elements with a `type` [attribute value][] of `text`.
```